ProjectedFSLib.Managed.API/NotificationMapping.h

```diff
@@ -138,11 +149,10 @@ namespace ProjFS {
         : notificationMask(notificationMask)
         , notificationRoot(notificationRoot)
     {
-        // The underlying native API expects that the virtualization root is specified as an empty
-        // string.  If the caller provides ".", convert that to the correct form.
-        if (notificationRoot->Equals("."))
+        if (notificationRoot->StartsWith("."))

```

Comment:
Fails for folders that start with "." -- e.g. ".git".
```suggestion
        if (notificationRoot =="." || notificationRoot->StartsWith(".\\"))
```

Reply:
Good catch, thanks.